modules/rosa-hcp-upgrading-cli-tutorial.adoc

```diff
@@ -74,6 +74,31 @@ VERSION  NOTES
 The latest available update for machine pools is limited to the current current version of the control plane. Ensure your control plane is up to date first.
 ====
 
+
+. Check the node drain grace period for your machine pools by running the following command:
```

Comment:
I think the place is not good.The node drain grace period is just an attribute for nodepool.
It is not the same level for cluster upgrade